.azure-pipelines/build-docker-sonic-vs-template.yml

```diff
@@ -14,15 +14,51 @@ parameters:
 - name: swss_artifact_name
   type: string
 
-- name: sairedis_artifact_name
+- name: swss_common_artifact_name
   type: string
 
-- name: swss_common_artifact_name
+- name: swss_common_branch
   type: string
+  default: '$(BUILD_BRANCH)'
 
 - name: artifact_name
   type: string
 
+- name: buildimage_artifact_name
+  type: string
+  default: 'sonic-buildimage.vs'
+
+- name: buildimage_artifact_project
+  type: string
+  default: 'build'
+
+- name: buildimage_artifact_pipeline
+  type: string
+  default: 'Azure.sonic-buildimage.official.vs'
+
+- name: buildimage_artifact_branch
+  type: string
+  default: '$(BUILD_BRANCH)'
+
+- name: sairedis_artifact_name
+  type: string
+
+- name: sairedis_artifact_project
+  type: string
+  default: 'build'
+
+- name: sairedis_artifact_pipeline
+  type: string
+  default: 'Azure.sonic-sairedis'
+
+- name: sairedis_artifact_branch
+  type: string
+  default: '$(BUILD_BRANCH)'
+
+- name: sairedis_artifact_pattern
+  type: string
+  default: '**'
+
```

Comment:
I don't understand why pipeline template need to change.
buildimage_artifact_branch need to be equal with sairedis_artifact_branch to avoid code unmatch.
I don't know why we need these parameters

Reply:
I can address this in a separate PR, we need to get this one merged with priority